pages/common/parquet-tools.md

```diff
@@ -0,0 +1,36 @@
+# parquet-tools
+
+> A tool to show, inspect and manipulate Parquet file.
+> More information: <https://github.com/apache/parquet-mr/tree/master/parquet-tools>.
+
+- Print the content of a Parquet file:
+
+`parquet-tools cat {{path/to/parquet}}`
```

Comment:
I installed the `parquet-tools` package in Arch Linux and the commands are in the top level of Bash:
```suggestion
`parquet-cat {{path/to/parquet}}`
```
the same for: `parquet-dump`, `parquet-head`, `parquet-merge`, `parquet-meta`, `parquet-rowcount` and `parquet-schema`.

Reply:
@navarroaxel 
Thank you for your suggestions.

I installed parquet-tools package in MacOS X using homebrew.
I can't find these commands like parquet-cat.

These commands are only wrapper command for jar file. 
You can call java -jar ... instead of parquet-tools (https://github.com/apache/parquet-mr/tree/master/parquet-tools#run-locally).
It means that we can use another wrapper like `parquet-cat`  or `parquet-tools cat`. 
And parquet-tools is the most major wrapper tools since I can't find parquet-cat in stackoverflow (see, https://stackoverflow.com/search?q=parquet-cat).

So I think my original code is better.